CHANGES.md

```diff
@@ -6,6 +6,10 @@
 * update FastAPI requirements
 * update minimal python version to 3.6
 
+**breaking change**
+
+* renamed `OptionalHeader`, `MimeTypes` and `ImageDrivers` to the singular form.
```

Comment:
```suggestion
* renamed `OptionalHeaders`, `MimeTypes` and `ImageDrivers` to the singular form.
```

Reply:
Also nit: the term "MIME Type" is deprecated in favor of "Media Type" I believe

Reply:
let's go all in